test/AnalyzerUser.targets

```diff
@@ -0,0 +1,16 @@
+<Project>
+  <ItemGroup>
+    <AnalyzerProjectReference Include="$(RepoRootPath)src\Microsoft.ServiceHub.Analyzers\Microsoft.ServiceHub.Analyzers.csproj" />
+    <AnalyzerProjectReference Include="$(RepoRootPath)src\Microsoft.ServiceHub.Analyzers.CSharp\Microsoft.ServiceHub.Analyzers.CSharp.csproj" />
+    <AnalyzerProjectReference Include="$(RepoRootPath)src\Microsoft.ServiceHub.Analyzers.CodeFixes\Microsoft.ServiceHub.Analyzers.CodeFixes.csproj" />
+    <ProjectReference Include="@(AnalyzerProjectReference)">
+      <ReferenceOutputAssembly>false</ReferenceOutputAssembly>
+      <OutputItemType>AnalyzerAssembly</OutputItemType>
```

Comment:
Is there a difference between this way and specifying <OutputItemType>Analyzer</OutputItemType> instead?